pkg/aws/ec2.go

```diff
@@ -119,6 +133,29 @@ func AssertEC2VolumeEncrypted(t *testing.T, ctx context.Context, client EC2Clien
 	assert.True(t, deviceEncrypted, "Volume with device ID '%s' for instance '%s' was not encrypted.", input.DeviceID, input.InstanceID)
 }
 
+// AssertVolumeType asserts the right volume type & associated throughput
+func AssertEC2VolumeType(t *testing.T, ctx context.Context, client EC2Client, input AssertEC2VolumeTypeInput) {
+
+	instance, err := getEC2InstanceByInstanceIDE(ctx, client, input.InstanceID)
+	assert.Nil(t, err)
+
+	for _, v := range instance.BlockDeviceMappings {
+		if *v.DeviceName == input.DeviceID {
+			volume, err := getEC2VolumeByVolumeIDE(ctx, client, *v.Ebs.VolumeId)
+			assert.Nil(t, err)
```

Comment:
**Suggestion:** Rather than `assert` here, have you considered using `require` or using the combo of `t.Error(err)` and `continue`? As it stands, the method won't exit if an error occurs retrieving the EC2 instance details, which I can see leading to some vague panic errors later when the `instance` variable is `nil`.

Reply:
Addressed this.